src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -507,8 +535,10 @@ def find_roots(frame_filename: FrameInfo, source_path: str) -> tuple[str, str]:
         return (stack_root, "")
     elif source_path.endswith(stack_path):  # "Packaged" logic
         source_prefix = source_path.rpartition(stack_path)[0]
-        package_dir = stack_path.split("/")[0]
```

Comment:
In Python, it has always been a single word representing the package name (e.g. `sentry`) while in Java it is multiple words (e.g. `io.sentry.some_app_name`), thus, we need to change it.